cosima_cookbook/explore.py

```diff
@@ -1033,14 +1046,23 @@ def _load_data(self, b):
         )
 
         try:
-            self._loaded_data = querying.getvar(
-                self.experiment_name,
-                varname,
-                self.de.session,
-                start_time=str(start_time),
-                end_time=str(end_time),
-                frequency=frequency,
-            )
+            if frequency == 'static':
```

Comment:
I feel like this would be clearer with a single call to `getvar` and just providing `n` or `start_time/end_time` through kwargs?

Reply:
Yeah, being a bit lazy there. Your suggestion was a good one, as it led me to refactor that whole thing in a more consistent way. Tah.